src/lib.rs

```diff
@@ -12,7 +12,6 @@
 #![feature(compiler_builtins)]
 #![feature(core_intrinsics)]
 #![feature(naked_functions)]
-#![feature(i128_type)]
```

Comment:
You'd most likely have to do `#[cfg_attr(stage0, feature(i128_type))]` here, as on stage0 it is still needed.

Reply:
Hmm... It seemed to work fine when I made this change locally?